pkg/webhook/pod/pd_deleter.go

```diff
@@ -59,12 +62,12 @@ func (pc *PodAdmissionControl) admitDeletePdPods(pod *corev1.Pod, ownerStatefulS
 	// NotMember represents this pod is deleted from pd cluster or haven't register to pd cluster yet.
 	// We should ensure this pd pod wouldn't be pd member any more.
 	if !isMember {
-		return pc.admitDeleteNonPDMemberPod(IsDeferDeleting, isInOrdinal, pod, ownerStatefulSet, tc, pdClient, ordinal)
+		return pc.admitDeleteNonPDMemberPod(IsDeferDeleting, isInOrdinal, payload, ordinal)
```

Comment:
Actually, all the other arguments except for payload can be calculated within the function with values of payload, right?
It's the same for the following functions.

Reply:
updated.